contrib_bots/lib/DefineBot/docs.md

```diff
@@ -0,0 +1,18 @@
+# DictBot
+
+This is a bot that defines a word that the user inputs.
+Whenever the user inputs a message starting with '@define',The bot defines the 
```

Comment:
This line should read: `Whenever the user inputs a message starting with '@define', the bot defines the`
(Note: space after comma, lower case t in the)